directories/src/redux/services/broker-node.js

```diff
@@ -42,4 +103,4 @@ export default {
   completeBrokerNodeAddressPoW,
   requestGenesisHashPoW,
   completeGenesisHashPoW
-};
```

Comment:
you forgot to export `claimTreasure`

Reply:
its because we are not using it at the moment